oid4vc/oid4vc/routes.py

```diff
@@ -189,6 +189,9 @@ async def exchange_create(request: web.Request):
 
     did = body.pop("did", None)
     verification_method = body.pop("verification_method", None)
```

Comment:
popping is no longer necessary since we're not splatting `body` anymore. Let's change these to `get`s for the sake of consistency.